pentaho-karaf-assembly/src/main/filtered-resources/etc-server/custom.properties

```diff
@@ -118,7 +118,9 @@ org.osgi.framework.system.packages.extra= \
  org.owasp.encoder.*, \
  io.reactivex.*;version\="2.0.4", \
  org.pentaho.di.engine.*;version="${pdi.version}", \
- org.reactivestreams.*;version="1.0.0"
+ org.reactivestreams.*;version="1.0.0", \
+ org.pentaho.di.trans.dataservice.client.api.*;version="${pentaho-osgi-bundles.version}", \
+ org.pentaho.di.trans.dataservice.jdbc.api.*;version="${pentaho-osgi-bundles.version}"
```

Comment:
Versions in the `custom.properties` need to comply with the OSGi version syntax. I believe that at the moment we don't have a problem but it's best to validate this with @jmcrfp

Reply:
That's the same property that is already being used extensively on that file. Either way I've pinged @jmcrfp about it...

Reply:
I've just confirmed.... that property already have the right format value.